### ISSUE TITLE
[Tips and Techniques Pages] Responsive Images

### ISSUE THREAD

**Issue by yatil** [w3c/wai-mobile-intro#10]
This page is not mentioning accessibility aspects. The resolution independency aspect is valid for desktop as well, most computers have high dpi numbers these days. The art direction use case is also dependend on the position of the image on a page and not only if it is mobile. The page also does not describe when which solution should be used.

I think we should remove this page.

**Comment by iamjolly** [comment]
I tend to agree with @yatil about removing this page. If the only WCAG guidance here is to adequately describe images, I think we don't need to dedicate a page for that in this developer intro. We may want to provide a brief mention to developers and link to other existing guidance on providing alt text.
